ujenkins/endpoints/builds.py

```diff
@@ -35,20 +64,36 @@ def get(self, name: str) -> List[dict]:
             ]
 
         Args:
-            name (str):
-                Job name or path (if in folder).
+            name (str): Job name or path (if in folder).
+            fields (Optional[Union[List[str], str]]): List of fields to return.
+                Possible values are mentioned in the available fields.
+                If empty, default fields will be returned.
+            start (int, optional): The start index of the builds to retrieve.
+                Used with the 'end' parameter to specify a range.
+                Defaults to None.
+            end (int, optional): The end index of the builds to retrieve.
+                Used with the 'start' parameter to specify a range.
+                Defaults to None.
 
         Returns:
-            List: list of build for specified job.
+            List[dict]: list of builds for specified job.
         """
+
+        page_string = ''
```

Comment:
`slice` variable name is more clear IMHO

Reply:
slice is a python standard function isn't it? https://www.w3schools.com/python/ref_func_slice.asp 
I stay away from overloading like that because I have been burned in the past, I don't mind either way though if you think it is O.K.

As an alternative how do you feel about page_slice or response_slice? Let me know your thoughts

Reply:
Yes, of course `slice` is reserved keyword, I meant to change your variable name to something more clear, maybe `pagination`?